test/simplelabel/simplelabel.str

```diff
@@ -11,82 +11,34 @@ frame: 0 {
    variable: seg {
       type: TYPE_SEG;
       conditionalparents: nil using DenseCPT("start_seg");
+      % Line below defines labels
```

Comment:
Remove "line below" and mention that is uses strings from the following name collection in input.master. For both comments in this .str file.